.github/CODEOWNERS

```
@@ -45,6 +45,7 @@ sycl/plugins/esimd_emulator/ @intel/dpcpp-esimd-reviewers
 
 # CUDA plugin
 sycl/plugins/cuda/ @intel/llvm-reviewers-cuda
+sycl/plugins/unified_runtime/ur/adapters/cuda/ @intel/llvm-reviewers-cuda
```

```suggestion
sycl/plugins/**/cuda/ @intel/llvm-reviewers-cuda
```
Will this work for you? I assume we name directories responsibly. :)
Basically, this says that any directory named `cuda` inside `sycl/plugins` is owned by @intel/llvm-reviewers-cuda team.

---

Yes, that makes sense, thanks!